services/airlock/README.md

```diff
@@ -0,0 +1,28 @@
+# airlock
+
+This directory contains the configuration for the airlock service. It
+runs as a docker container via `docker compose`, and is automatically updated to
+the :latest tag.
+
+The container is always named `airlock`, and can be managed using
+normal docker tools or via `docker compose`.
+
+# Configuration
+
+airlock shares the same configuration as jobrunner, which currently lives
+in `/home/opensafely/config/*.env`. In particular, it uses `AIRLOCK_API_TOKEN` to
+sign and validate requests. This currently duplicates `JOB_SERVER_TOKEN`.
```

Comment:
Yeah, I think we should rename the env var in airlock to JOB_SERVER_TOKEN, as its shared between apps and pre-exists.

Reply:
agree, I will open a ticket on the airlock repo - what should we do here & now, though? do you want to do something like 

```bash
AIRLOCK_API_TOKEN="$JOB_SERVER_TOKEN"
```

or merge as-is for now?

Reply:
gonna merge as-is